### DISCUSSION TITLE
Running blatann with python 3.11

### DISCUSSION THREAD

**Discussion by alexdonati** [ThomasGerstenberg/blatann#177] [Q&A]
Hi, 
there is any chance to install blatann with python > 3.10 ??

Limitation is starting to be bottleneck.

thanks
Alessandro

**Comment by ThomasGerstenberg** [comment]
Yes, however it requires a bit of a workaround now that the mainline `pc-ble-driver-py` repository is no longer active.

There are some forks of that repo that are building the library for 3.11, such as https://github.com/mickael9/pc-ble-driver-py. You should be able to download and install the wheel from their latest CI job ([here](https://github.com/mickael9/pc-ble-driver-py/actions/runs/8987385708)), then pip install blatann afterwards and it should work.

Note that I haven't tested this myself yet. I'm also looking into bundling parts of that library into blatann (the parts I need) and remove the dependency altogether

**Reply by ThomasGerstenberg** [reply]
Update: I've forked pc-ble-driver-py and updated it so a wheel is available for Python 3.11 (also ARM MacOS). The wheels can be downloaded from this CI job: https://github.com/ThomasGerstenberg/pc-ble-driver-py/actions/runs/10399393894
